test_files/test_bld.asm

```diff
@@ -0,0 +1,13 @@
+;-- Setting bits to true which are already true.
+ldi r31, 0xFF
+set
+bld r31, 6
+bld r31, 5
+bld r31, 4
```

Comment:
These basically don't do anything. As the comment there says, you set all bits in r31 to 1, you set T to 1 then you try to set bits 6, 5 and 4 in r31 to 1, but they are already one. Are you trying to check that nothing weird happens when you try to set a specific bit to 1 when it's already one? Then specify it clearly in the comment. 

If not, then what I would suggest doing is doing the same thing you did in lines 9 through 13. Load the value 0x0F into the register R31. Half of them are 0, half 1. 

First you set the T flag to 1, then you use BLD to set the bits 0, 2, 4, 6 for example. This will show you the behaviour of the instruction on bits that are 1 and on bits that are 0, exploring every possibility.

Explain the rationale in a comment in the code for lines 2-6, and for lines 9-13.